src/components/Modal/Modal.vue

```diff
@@ -201,7 +207,6 @@ export default {
 	height: 100%;
 	width: 100%;
 	.modal-container {
-		width: 70%;
 		max-width: 900px;
```

Comment:
Shouldn't we also remove the max-width/height then and leave this up to the implementing apps?

Reply:
No, this is actually nice to have a maximum standard. Just not a default size in case we want things smaller. But the maximum is a good thing to have :)

Reply:
Ok fine by me then :+1: